src/ducks/Signals/signalFormManager/SignalMaster.js

```diff
@@ -3,22 +3,22 @@ import { push } from 'react-router-redux'
 import { graphql } from 'react-apollo'
 import { compose } from 'recompose'
 import { connect } from 'react-redux'
-import { createTrigger, toggleTrigger, updateTrigger } from '../Redux/actions'
+import { createTrigger, updateTrigger } from '../requx/actions'
```

Comment:
Probably misspelled `redux` in a folder name

Reply:
I wants group them by types. Because all files existsed in one folder. And I moved actions/reducers into redux folder.

Reply:
Please, read my comment again. You moved them to `requx` folder, not `redux`